src/components/RichSelect/__stories__/Disabled.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import { ComponentStory } from '@storybook/react'
+import RichSelect from '..'
+
+export const Disabled: ComponentStory<typeof RichSelect> = ({ ...props }) => (
+  <RichSelect name="disabled" disabled value="a" {...props}>
```

Comment:
for these, can we not set Disabled.args with children for options and juste disabled: true ?